### PR TITLE
Overhaul "Install" instructions

### LINKED ISSUE #51: Mention the rgbds docker image
As we're [now publishing ](https://github.com/gbdev/rgbds/pkgs/container/rgbds) the RGBDS container package, the docker option should be mentioned in the Install page

### PR DESCRIPTION
**Pull request by avivace**
I wasn't quick enough to review #47, so let me try again.

The install page suffers from a lot of problems right now, making the whole thing unreadable and inaccessible to the average target of "new users of RGBDS".

Let's try to review the criteria:

- [x] Keep the details to the minimum. We are not interesting on explaining the intricacies of installing on Windows or in which way different ways of installing on Windows are related.
- [x] Our primary dev environment (and the largest RGBDS user quota) target is "Linux". We have pre-built binaries for amd64 on the CI (?). Until we start packaging it for main distributions, that should be the primary suggested way to obtain the binaries. End of the story.
- [x] Reading the install page is incredibly complex right now and leads to so many choices that it makes me want to give up just looking at all the branches and choices I have to make.

Feel free to discuss any of these points. In the meantime, I started an overhaul:

- Removed mention of rgbds2 package. Why would someone be interested in it? It will just make the explanation and the options more complex for new users. If I need an old version, I'll look up for an old release, no need to imply *3* choices in the Arch Linux tab. it's not an alternative/LTS branch receiving updates, so it's not like I need to subscribe to a rgbds2 package.
- Made a generic "Linux" tab as the default. Go to releases, pick a pre-built binary for the arch you need. You're done.

Fixes #51.

### CODE REVIEW

**Comment by ISSOtm** [comment]
Are you sure the largest portion of our user base is on Linux? Lacking any better info (people *will* get mad if we add any telemetry), I've been referencing the [Sublime Text extension](https://packagecontrol.io/packages/RGBDS)'s stats, which are approximately 50% Windows, 25% macOS, 25% Linux.

As far as I'm aware, distributing raw Linux binaries is a bad idea because they'll be incompatible across distributions? I will look up info on that.

**Comment by avivace** [comment]
Let's also remember to mention rgbenv

**Comment by avivace** [comment]
This is currently blocked by https://github.com/gbdev/rgbds/pull/1148 not being applied to old releases, so we don't have the Linux build until we release

**Comment by avivace** [comment]
https://github.com/gbdev/rgbds-www/issues/51 could also be fixed here

**Comment by ISSOtm** [comment]
I'm not sure Docker would be much use for installing RGBDS?

File: src/pages/install/index.md [hunk 1]
```diff
@@ -4,25 +4,25 @@ import TabItem from '@theme/TabItem';
...
 If your OS isn’t listed below, try finding `rgbds` in your package manager—refer to your OS' documentation for help.
 If you still can’t find RGBDS, or the specific version you are looking for is unavailable, you will have to [compile it from source](#building-from-source).
 
+
 <Tabs>
-<TabItem value="windows" label="Windows">
+<TabItem value="linux" label="Linux">
 
-The install instructions change a bit depending on the environment you wish to use RGBDS with.
+TO DO
```
Comment (ISSOtm) [inline src/pages/install/index.md:19]: We should suggest, in order:

- Using the distribution's package manager, if available (providing some examples?)
- Using our pre-built binaries
- Docker?
- Compiling from source
- ??? Anything I missed?

File: src/pages/install/index.md [hunk 2]
```diff
@@ -4,25 +4,25 @@ import TabItem from '@theme/TabItem';
...
 
-The "plain Windows" instructions may also work for some environments listed further below, since most of them pick up Windows' `PATH`.
-However, if you have such an environment, we recommend you follow its specific install instructions instead.
+<TabItem value="windows" label="Windows">
 
-:::
+The install instructions change a bit depending on the environment you wish to use RGBDS with.
 
 <details>
 <summary>"Plain" Windows (CMD, PowerShell)</summary>
```
Comment (ISSOtm) [inline src/pages/install/index.md:28]: Actually, the WSL instructions should instruct to refer to the Linux instructions for the corresponding Linux distribution (noting that if you don't know, it's probably Ubuntu).

I'm thinking we should organise the tabs something like `WSL`, then `MSYS2 or Cygwin`, and finally `none of these`. That should make what the last one is about, clearer. I think.

File: src/pages/install/index.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ On WSL, the default package manager (to install any build prerequisites) is `apt
 </TabItem>
 <TabItem value="arch" label="Arch Linux">
```
Comment (ISSOtm) [inline src/pages/install/index.md:81]: Let's move all distro-specific instructions to the "Linux" tab, right?

**Review by ISSOtm — CHANGES_REQUESTED** [review]

File: src/pages/install/index.md (hunk 2 above)
Comment (avivace) [reply src/pages/install/index.md:49]: This seems a good idea, although I have no experience whatsoever on Windows so if you have any specific ideas on how to reorganize the Windows tab feel free to push

**Comment by pinobatch** [comment]
It has been suggested to use https://nightly.link/ to solve the [lack of guest downloads for `master`](https://github.com/actions/upload-artifact/issues/51).

**Comment by avivace** [comment]
@ISSOtm

> I'm not sure Docker would be much use for installing RGBDS?

The container image provides a very quick way, literally a oneliner (if you have a container runtime installer ofc) to obtain a linux environment able to run AND build RGBDS (the version you're checking out, but potentially any other).

It may also be useful as a "base" image for pipelines.

Anyway, we'll have some stats in a while on how popular/really used really is, so I'm okay in removing it from the options if it we understand is not really used..

---

Another thing, are we really sure we want the Tabs navigation here? Now the instructions were slimmed down, they seem to amount just 4-5 lines per platform so I'm now sure if requiring additional clicks is really useful? 

---

@PinoBatch where should the nightly link https://nightly.link/gbdev/rgbds/workflows/testing/master go? In the "Install master?" (https://rgbds.gbdev.io/install/master) (which could also be renamed to "Canary" tbf, it's clearer

File: src/pages/install/index.md [hunk 4]
```diff
@@ -4,25 +4,46 @@ import TabItem from '@theme/TabItem';
 
 # Installing RGBDS
 
-Most people will want to use a release (the latest the bestest!), but if you are willing to help us test new features, you may consider [using a development version](/install/master).
+Most people will want to use a stable release (the more recent the better). 
 
-The two main options to install a release of RGBDS are to download pre-built executables, or to [build from source](/install/source).
-Below is a list of <abbr title="Operating System">OS</abbr>es for which we know pre-built executables are available.
+Below is a list of Operative Systems for which pre-built executables are available. If none of these options fits your needs, you can [build from source](/install/source).
```
Comment (ISSOtm) [inline src/pages/install/index.md:10]:
```suggestion
Below is a list of OSes for which pre-built executables are available.
If none of these options fits your needs, you can [build from source](./source.md).
```
We can use `<abbr title="Operating Systems">` to explain "OSes" instead, if desired.
Additionally, linking to the Markdown file makes it checked by Docusaurus, which improves reliability.

File: src/pages/install/index.md [hunk 5]
```diff
@@ -4,25 +4,46 @@ import TabItem from '@theme/TabItem';
...
-If you still can’t find RGBDS, or the specific version you are looking for is unavailable, you will have to [compile it from source](#building-from-source).
 
 <Tabs>
-<TabItem value="windows" label="Windows">
+<TabItem value="linux" label="Linux">
 
-The install instructions change a bit depending on the environment you wish to use RGBDS with.
+Linux x86_64 ready binaries are available in our [Releases](https://github.com/gbdev/rgbds/releases) page.
+
+##### Arch
```
Comment (ISSOtm) [inline src/pages/install/index.md:17]:
```suggestion
## Arch
```
We're only at heading level 2 here; level 3 might be appropriate too.

File: src/pages/install/index.md [hunk 6]
```diff
@@ -78,30 +99,24 @@ On WSL, the default package manager (to install any build prerequisites) is `apt
...
-RGBDS is available on [Homebrew](https://brew.sh) as the [`rgbds` package](https://formulae.brew.sh/formula/rgbds).
+<br />
 
-```bash
-brew install rgbds
-```
+### Installing a development version
 
-You can also install the [`master` branch](/docs/master) by passing the `--HEAD` flag:
+If you are willing to help us test new features, consider [using a development version](/install/master).
```
Comment (ISSOtm) [inline src/pages/install/index.md:118]:
```suggestion
If you are willing to help us test new features, consider [using a development version](./master.md).
```

File: src/pages/install/index.md [hunk 7]
```diff
@@ -78,30 +99,24 @@ On WSL, the default package manager (to install any build prerequisites) is `apt
...
-<TabItem value="macos" label="macOS">
+</Tabs>
 
-RGBDS is available on [Homebrew](https://brew.sh) as the [`rgbds` package](https://formulae.brew.sh/formula/rgbds).
+<br />
 
-```bash
-brew install rgbds
-```
+### Installing a development version
```
Comment (ISSOtm) [inline src/pages/install/index.md:116]:
```suggestion
## Installing a development version
```

File: src/pages/install/index.md [hunk 8]
```diff
@@ -78,30 +99,24 @@ On WSL, the default package manager (to install any build prerequisites) is `apt
...
 
-You can also install the [`master` branch](/docs/master) by passing the `--HEAD` flag:
+If you are willing to help us test new features, consider [using a development version](/install/master).
 
-```bash
-brew install rgbds --HEAD
-```
+### Managing multiple versions
```
Comment (ISSOtm) [inline src/pages/install/index.md:120]:
```suggestion
## Managing multiple versions
```

File: src/pages/install/index.md [hunk 9]
```diff
@@ -4,25 +4,46 @@ import TabItem from '@theme/TabItem';
...
+Linux x86_64 ready binaries are available in our [Releases](https://github.com/gbdev/rgbds/releases) page.
+
+##### Arch
+
+RGBDS is available in the official Arch Linux repositories as [`community/rgbds`](https://www.archlinux.org/packages/community/x86_64/rgbds/); you can also get the [latest master](/docs/master) via the [`rgbds-git`](https://aur.archlinux.org/packages/rgbds-git) AUR package.
+
+```bash
+pacman -S rgbds
+```
+
```
Comment (ISSOtm) [inline src/pages/install/index.md:24]:
```suggestion

```

File: src/pages/install/index.md (hunk 2 above)
Comment (ISSOtm) [reply src/pages/install/index.md:49]: I'm not a Windows user anymore, but I think I have some ideas. I'll do that then.

**Review by ISSOtm — CHANGES_REQUESTED** [review]

File: src/pages/install/index.md (hunk 2 above)
Comment (ISSOtm) [reply src/pages/install/index.md]: OK, done!

File: src/pages/install/index.md (hunk 4 above)
Comment (avivace) [reply src/pages/install/index.md:10]: Agree!
